src/Artsy/Router/Utils/Route.tsx

```diff
@@ -50,11 +55,15 @@ function createRender({ prerender, render, renderFetched }) {
 
     // This should only ever show when doing client-side routing.
     if (!props) {
-      return (
-        <SpinnerContainer className={LoadingClassName}>
-          <Spinner />
-        </SpinnerContainer>
-      )
+      if (showPreloader) {
```

Comment:
If for whatever reason a dev doesn't want to show the default preloader on route change (in favor of some kind of custom loader via SystemContext) they can pass `showPreloader: false` into their route.

Reply:
Do we actually have a use-case for this right now?

Reply:
Not at the moment. Similar to the other comment, can remove until this is needed 👍